index.html

```diff
@@ -5,9 +5,53 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
-    
+    <title>Morning exercise</title>
   </head>
   <body>
-
+    <header>
+      <h1>Incremental and Decremental Support Vector Machine Learning</h1>
+      <hr />
+    </header>
+    <section>
+      <p>
+        <strong>Gert Cauwenberghs</strong> <br />
+        CLSP, ECE Dept. <br />
+        Johns Hopkins University <br />
+        Baltimor, MD 21218 <br />
+        <em>gert@jhu.edu</em>
+      </p>
+    </section>
+    <main>
+      <h2>Abstract</h2>
+      <p>
+        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quod quae
+        tenetur ex obcaecati dicta? Suscipit vero, repellendus voluptates
+        incidunt reiciendis aut quae distinctio beatae voluptatum officia. Hic
+        aliquam itaque tenetur?
+      </p>
+      <ol>
+        <li>
+          <strong>Introduction</strong>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates
+            cupiditate velit dolores hic libero perspiciatis error minima fuga?
+            Vel suscipit dolor mollitia perspiciatis deserunt, debitis quibusdam
+            eaque qui inventore eligendi?
+          </p>
+          <img
+            src="https://docs.opencv.org/2.4/_images/optimal-hyperplane.png"
+            alt="Graph shwoing the optimal hyperplane"
+          />
+        </li>
+        <li></li>
+      </ol>
+      <nav>
+        <a href="https://en.wikipedia.org/wiki/Support_vector_machine"
+          >Go to Wikipedia to know more</a
+        >
```

Comment:
The formatting has gone a little wacky here but other than that, this is a very well structured HTML document. I'd add a bit of spacing between sections, e.g.

```html
    <header>
      <h1>Incremental and Decremental Support Vector Machine Learning</h1>
      <hr />
    </header>
    
    <section>
```